test/integration/stats_integration_test.cc

```diff
@@ -355,7 +363,9 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithRealSymbolTable) {
   // If you encounter a failure here, please see
   // https://github.com/envoyproxy/envoy/blob/master/source/docs/stats.md#stats-memory-tests
   // for details on how to fix.
-  EXPECT_MEMORY_EQ(m_per_cluster, 36827);
+  if (ip_version_ != Network::Address::IpVersion::v6) {
```

Comment:
maybe add a comment explaining why we only check for ipv6?

Reply:
done